### PR TITLE
envoy: build on ARM

### PR DESCRIPTION
**Pull request by carlocab**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

### CODE REVIEW

**Comment by carlocab** [comment]
Ah, right. Still no good, as the build tries to download a `bazel` that is too old.

**Comment by bayandin** [comment]
> Ah, right. Still no good, as the build tries to download a `bazel` that is too old.

Can we use homebrewed `bazel` as we do for `libtensorflow`?

**Comment by carlocab** [comment]
I don't really know how `bazel` works, so I've just updated this PR with a naive approach... Maybe it'll work?

**Comment by bayandin** [comment]
> I don't really know how `bazel` works, so I've just updated this PR with a naive approach... Maybe it'll work?

Looks  good, just maybe need to add
```ruby
(buildpath / ".bazelversion").atomic_write Formula["bazel"].version
```

File: Formula/envoy.rb [hunk 1]
```diff
@@ -22,16 +22,16 @@ class Envoy < Formula
   depends_on macos: :catalina
 
   def install
-    args = %w[
+    args = %W[
```
Comment (bayandin) [inline Formula/envoy.rb:25]:
```suggestion
    (buildpath / ".bazelversion").atomic_write Formula["bazel"].version
    args = %W[
```

**Comment by bayandin** [comment]
Nah, still no luck on ARM
```
Current running Bazel is ahead of bazel-toolchains repo. Please update your pin to bazel-toolchains repo in your WORKSPACE file.
```

**Pull request closed without merging**